modular_skyrat/modules/sec_haul/code/guns/bullets.dm

```diff
@@ -54,45 +54,45 @@
 /obj/projectile/bullet/advanced/b6mm/ihdf
 	name = "6.3mm fragmentation pellet"
```

Comment:
```suggestion
	name = "6mm fragmentation pellet"
```

Reply:
Jesus I didn't even notice that the pathing on that was wrong for whatever reason